pages/common/mat2.md

```diff
@@ -0,0 +1,23 @@
+# mat2
+
+> Anonymise various file formats by removing metadata. More information: <https://0xacab.org/jvoisin/mat2>.
+
+- List supported file formats:
+
+`mat2 --list`
+
+- Remove metadata from a file:
+
+`mat2 {{file_path}}`
```

Comment:
```suggestion
`mat2 {{path/to/file}}`
```
Could you please update `file_path` to `path/to/file` to be consistent with other pages? Same goes for other examples below this one.